src/Files.App/Views/Properties/MainPropertiesPage.xaml.cs

```diff
@@ -308,7 +309,7 @@ private void Page_KeyDown(object sender, KeyRoutedEventArgs e)
 		private void ClosePage()
 		{
 			if (_usingWinUI)
-				AppWindow.Destroy();
+				Window.Close(); // AppWindow.Destroy() doesn't seem to work well. (#11461)
```

Comment:
Can you move the comment above the if statement?

Reply:
Could be under the if statement, main thing is to put it on top of Window.Close..

Reply:
OK. Moved.